### DISCUSSION TITLE
How to compile vcpkg on Fedora on arm64

### DISCUSSION THREAD

**Discussion by sandeep-gh** [microsoft/vcpkg#28038] [Q&A]
I am following the steps mentioned here: https://github.com/microsoft/vcpkg/issues/3647#issuecomment-658714967
to build vcpkg on Fedora on Raspberry pi. 
I did the following step:
`export VCPKG_DEFAULT_TRIPLET=arm64-linux`  
`export VCPKG_FORCE_SYSTEM_BINARIES=1`  
`export VCPKG_USE_SYSTEM_BINARIES=1`  
`./bootstrap-vcpkg.sh -useSystemBinaries` .

Then created vcpkg/triplets/arm64-linux.cmake with requisite entries. 

Then ran the Cmake build command as follows:

`cmake -B build -S vcpkg/buildtrees/_vcpkg/src/vcpkg-tool-2021-11-24/ -DVCPKG_TARGET_TRIPLET=arm64-linux -DCMAKE_TOOLCHAIN_FILE=vcpkg/scripts/buildsystems/vcpkg.cmake`

I am not sure if I am giving the right source directory, if not all Cmake build dependencies are installed. But I get error:

`
CMake Error at /usr/share/cmake/Modules/CMakeDetermineSystem.cmake:130 (message):
  Could not find toolchain file: vcpkg/scripts/buildsystems/vcpkg.cmake
Call Stack (most recent call first):
  CMakeLists.txt:37 (project)

CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!`

**Comment by JackBoosY** [comment]
Maybe you should use absolute path instead?

**Comment by masx200** [comment]
https://github.com/microsoft/vcpkg/discussions/24173
